clusterHost.py

```diff
@@ -89,10 +89,11 @@ def _create_k8s_nodes(configs: list[NodeConfig]) -> list[ClusterNode]:
     def _k8s_nodes(self) -> list[ClusterNode]:
         return self.k8s_master_nodes + self.k8s_worker_nodes
 
-    def _ensure_images(self, local_iso_path: str, infra_env: str, nodes: list[ClusterNode]) -> None:
+    def ensure_images(self, local_iso_path: str, infra_env: str, masters: bool) -> None:
         if not self.hosts_vms:
             return
 
+        nodes = self.k8s_master_nodes if masters else self.k8s_worker_nodes
```

Comment:
Pass in which nodes to check from the caller instead of passing in a bool that is resolved.

Reply:
Done